pages/whats-new.js

```diff
@@ -27,26 +27,25 @@ export async function getServerSideProps(context) {
       }
     };
   } catch (err) {
-    notify(err?.response?.data?.message ?? err?.message, 'error');
-    
+    notify(err?.response?.data?.message ?? err?.message, "error");
+
     return {
       props: {
         blogs: []
       }
-    }
+    };
   }
 }
 
-export default function whatsNew({blogs}) {
-  console.log(blogs)
+export default function whatsNew({ blogs }) {
   return (
     <>
       <Head>
         <title> What's new | Nullcast</title>
       </Head>
       <SiteHeader />
       <WhatsNewSpotlight />
-     {blogs?.length > 0 && <WhatsNewPosts blogs={blogs} />}
```

Comment:
why removed this condition, check the value in blogs, if its empty correct the probs